src/components/guideGrid.tsx

```diff
@@ -11,14 +11,14 @@ type Props = {
 
 export function GuideGrid({platform, className}: Props) {
   const [currentPlatform] = usePlatform(platform);
```

Comment:
🤔 a little confusing that `usePlatform` might return a guide, but I guess that's just the data model?